projects/solar/inc/solar_config.h

```diff
@@ -29,6 +29,8 @@ const CanSettings *config_get_can_settings(void);
 
 const GpioAddress *config_get_drv120_relay_pin(void);
```

Comment:
Could you rename `config_get_drv120_relay_pin` to `config_get_drv120_enable_pin` since it's the enable pin, not the relay pin?